Doc/library/ast.rst

```diff
@@ -584,14 +541,11 @@ Expressions
 
    .. doctest::
 
-        >>> print(ast.dump(ast.parse("(x := 4)"), indent=4))
-        Module(
-            body=[
-                Expr(
-                    value=NamedExpr(
-                        target=Name(id='x', ctx=Store()),
-                        value=Constant(value=4, kind=None)))],
-            type_ignores=[])
+        >>> print(ast.dump(ast.parse("(x := 4)", mode="eval"), indent=4))
```

Comment:
Parenthesis are not necessary.

Reply:
With `eval` they were needed. We would need to drop "eval" if we don't want them, no?

Reply:
Hummmm..... I may be missing something:

```
>>> print(ast.dump(ast.parse('x := 4', mode='eval'), indent=4))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/pablogsal/github/python/master/Lib/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 1
    x := 4
      ^
SyntaxError: invalid syntax
```

Reply:
You are right. I thought parenthesis are necessary in the "exec" mode but not in "eval" mode.